### DISCUSSION TITLE
Login, logout and edit push functions are now available

### DISCUSSION THREAD

**Discussion by Frederisk** [Frederisk/Wikitext-VSCode-Extension#27] [Show and tell]
@duyuang:

Login, logout and edit push functions are now available (after version 3.0.0).

At the same time, I noticed that [the original expansion](https://github.com/daflyinbed/ewiv) has been sealed and removed from the market.

**Comment by DDEle** [comment]
I'm not sure if this is specific to Fandom users, but if anyone met problems logging in like me, try using bot username and password (Special:BotPasswords) may help. Hope this helps.

**Reply by CxGrizzy** [reply]
Hello! I am having issues as well with using posting updates because of an invalid login. 

I've tried not only using just "fandom.com" but also "mywiki.fandom.com" and using what you said above and creating a bot within my wiki itself but still no luck. I was wondering if you solved this issue and if so what I could do to fix it? 

Thank you, Best regards.

**Reply by Frederisk** [reply]
It might help to check that your settings in the extension match the `/Special:Version` on the site.
